src/base/settingsstorage.cpp

```diff
@@ -82,7 +82,8 @@ namespace
             { "AddNewTorrentDialog/SavePathHistory", "TorrentAdditionDlg/save_path_history" },
             { "AddNewTorrentDialog/Enabled", "Preferences/Downloads/NewAdditionDialog" },
             { "AddNewTorrentDialog/TopLevel", "Preferences/Downloads/NewAdditionDialogFront" },
-            { "ExecutionLog/Enabled", "Preferences/ExecutionLog/enabled" }
+            { "ExecutionLog/Enabled", "Preferences/ExecutionLog/enabled" },
+            { "MainWindow/Notifications/Enabled", "Preferences/General/ProgramNotification" }
```

Comment:
"Notifications/Enabled" (without "MainWindow").

We discussed it with @evsh. 
@glassez:
>I agree with you partially. But...
1. Objective correspondence between settings names and names of the components/properties is better than subjective separation into groups.
2. qBittorrent configuration file is mostly implementation detail and it isn't a user configuration interface.

@evsh:
>Consider the following scenario, please:
The **Session** class is refactored and the **TorrentFileGuard** becomes unneeded. Will you break the configuration, add a migration code or leave `TorrentFileGuard/` key in the **Session** class?

I realized that we really don't have to associate the names of the settings with the names of the classes (this does not apply to those cases where we save its state). But the old way is also not good (using grouping from the settings window). Therefore, we must use a more abstract grouping based on the meaning of these options.